Source/WebCore/html/canvas/WebGLUtilities.h

```diff
@@ -53,14 +53,14 @@ class ScopedInspectorShaderProgramHighlight {
     void hideHighlight();
 
     struct {
-        GCGLfloat color[4];
-        GCGLenum equationRGB;
-        GCGLenum equationAlpha;
-        GCGLenum srcRGB;
-        GCGLenum dstRGB;
-        GCGLenum srcAlpha;
-        GCGLenum dstAlpha;
-        GCGLboolean enabled;
+        GCGLfloat color[4] { 0, 0, 0, 0 };
+        GCGLenum equationRGB { GraphicsContextGL::ZERO };
+        GCGLenum equationAlpha { GraphicsContextGL::ZERO };
+        GCGLenum srcRGB { GraphicsContextGL::ZERO };
+        GCGLenum dstRGB { GraphicsContextGL::ZERO };
+        GCGLenum srcAlpha { GraphicsContextGL::ZERO };
+        GCGLenum dstAlpha { GraphicsContextGL::ZERO };
```

Comment:
Wasn't sure if we should use `GraphicsContextGL::ZERO` or just `0` here.

Reply:
`GraphicsContextGL::NONE` might be appropriate to signal that the value is not initialised. In OpenGL, `srcRGB/srcAlpha` default to `GraphicsContextGL::ONE` and `dstRGB/dstAlpha` default to `GraphicsContextGL::ZERO`.

Reply:
So the proposal is this?

```suggestion
        GCGLenum equationRGB { GraphicsContextGL::NONE };
        GCGLenum equationAlpha { GraphicsContextGL::NONE };
        GCGLenum srcRGB { GraphicsContextGL::ONE };
        GCGLenum dstRGB { GraphicsContextGL::ZERO };
        GCGLenum srcAlpha { GraphicsContextGL::ONE };
        GCGLenum dstAlpha { GraphicsContextGL::ZERO };
```